### DISCUSSION TITLE
How to format the row with the fields names

### DISCUSSION THREAD

**Discussion by aborruso** [posit-dev/great-tables#116] [Q&A] [answered]
Hi,
I have built this first small example https://aborruso.quarto.pub/gt-0eb8/

I would like to have:

- all the fields names in bold (as the default quarto dataframe rendering);
- to know ho to differentiate the background of the row with the fields names, to differentiate it from the first row with the data.

Thank you for this great project!

![image](https://github.com/posit-dev/great-tables/assets/30607/eabafbc6-7be7-48b0-bfee-184095b5c90c)

**Comment by jrschueller** [comment, marked as answer]
I am experimenting with the `html()` [helper function](https://posit-dev.github.io/great-tables/reference/html.html#great_tables.html), and it looks like you can pass in whatever you wish. To drive the point home, here's one of the examples with the header swapped out with Comic Sans:

```from great_tables import GT, md, html
from great_tables.data import islands

islands_mini = islands.head(10)

(
    GT(islands_mini, rowname_col = "name")
    .tab_header(
        title = gt.html("<p style=\"font-family: Comic Sans MS; text-align: center\"><b>Large</b> landmasses of the world</p>"),
        subtitle = "The top ten largest are presented"
    )
    .tab_source_note(
        source_note = "Source: The World Almanac and Book of Facts, 1975, page 406."
    )
    .tab_source_note(
        source_note = md("Reference: McNeil, D. R. (1977) *Interactive Data Analysis*. Wiley.")
    )
    . tab_stubhead(label = "landmass")
)
```
![Screenshot 2024-01-29 115451](https://github.com/posit-dev/great-tables/assets/16596730/ca0c1d97-320e-4c97-afe3-54d68ebea032)
